cellprofiler/modules/editobjectsmanually.py

```diff
@@ -1,6 +1,16 @@
 # coding=utf-8
 
-"""
+import numpy as np
+
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
+import _help
```

Comment:
There are other imports after "See also".. Are all of them needed? I also added the .format(**{"HELP_ON_SAVING_OBJECTS": ....) after """. Let me know if it is not necessary. I assumed it is since we are adding it under See also.